README.adoc

```diff
@@ -1619,7 +1619,7 @@ end
 
 === Check Model Validity[[model-check-validity]]
 
-Add an example ensuring that the ``FactoryBot.create``d model is valid.
+Add an example ensuring that the ``FactoryBot.create`` model is valid.
```

Comment:
It was written this way to say "created", do you think this can be reworded to emphasize that the model is created using a factory?

Reply:
Oh, I got it. Thanks for your review, @pirj.
I've just reverted it and edited pull request's title also.

Reply:
Since you've noticed that the current wording is not perfect, do you mind taking a chance at fixing it? WDYT of
> Add an example ensuring that the model created with ``FactoryBot.create`` is valid.

Reply:
I've just edited it. Please see it again.